libs/entitlement/src/main/java/org/elasticsearch/entitlement/initialization/EntitlementInitialization.java

```diff
@@ -317,6 +319,16 @@ private static PolicyManager createPolicyManager() {
                 )
             )
         );
+
+        validateFilesEntitlements(
+            pluginPolicies,
```

Comment:
I considered to add validation for server and agent entitlements here too, but decided it's not worth it. Let me know if you thing those should be validated too.